packages/components/src/local/molecules/admin-message/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import moment from 'moment'
+import classNames from 'classnames'
+import Chip from '@material-ui/core/Chip'
+
+/* Import Types */
+import PropTypes from './types/props'
+
+/* Import Stylesheet */
+import styles from './styles.module.scss'
+
+/* Render component */
+export const AdminMessage: React.FC<PropTypes> = ({ message }: PropTypes) => {
+  const { details } = message
+  const { force, role, name } = details.from
+  const isOwnMessage = name === force
```

Comment:
@zukilover - I may be mistaken, but I think we should be matching the force of the logged in player.  I guess we can pass that in as props.  For the Story, I guess we could have checkbox knobs to allow switching between `White` and `Blue` force, which will switch which messages are _own_.

Reply:
You're correct. I must've missed on that own message checking logic. I'll update soon.

Reply:
After almost a year away from the project, you are completely forgiven 🤣

Reply:
Updated now :D